docs/usage/exporting_models.md

```diff
@@ -12,25 +12,28 @@ Arguments:
 * `by_alias`: whether field aliases should be used as keys in the returned dictionary; default `False`
 * `exclude_unset`: whether fields which were not explicitly set when creating the model should
   be excluded from the returned dictionary; default `False`.
-  Prior to **v1.0**, `exclude_unset` was known as `skip_defaults`; use of `skip_defaults` is now deprecated
 * `exclude_defaults`: whether fields which are equal to their default values (whether set or otherwise) should
   be excluded from the returned dictionary; default `False`
 * `exclude_none`: whether fields which are equal to `None` should be excluded from the returned dictionary; default
   `False`
+* `round_trip`: whether to enable serialization and deserialization round-trip support; default `False`
+* `warnings`: whether to log warnings when invalid fields are encountered; default `True`
```

Comment:
Should we link https://github.com/pydantic/pydantic-extra-types/pull/60 instead of writing those here?

Reply:
Sorry, Didn't get this.

Reply:
On the meeting (this morning), Samuel suggested to replace this section: https://docs.pydantic.dev/dev-v2/usage/model_config/#options; for the API reference: https://docs.pydantic.dev/dev-v2/api/config/#pydantic.config.ConfigDict .

What I'm suggesting here is to do the same.